tests/tests/regression_tests.rs

```diff
@@ -620,6 +620,7 @@ swf_tests! {
     (as3_istypelate_coerce, "avm2/istypelate_coerce", 1),
     (as3_class_cast_call, "avm2/class_cast_call", 1),
     (as3_class_supercalls_mismatched, "avm2/class_supercalls_mismatched", 1),
+    (as3_symbol_class_binary_data, "avm2/symbol_class_binary_data", 1),
```

Comment:
Putting a comment here because I can't mark the SWF.

Is there a source format that represents the SWF as a whole? I see the AS3 source but I'd like some way to regenerate the whole SWF if possible, `BinaryData` characters and all. If you used Animate to generate the test file, then this should include the FLA. If you used Flex or mxmlc, then there should be a project file (I think?); and rabcasm tests have it's own format.

If it's not possible to provide source for the SWF, that's fine too.

Reply:
I honestly couldn't manage to compile something with the binary tag using the mxmlc compiler (admittedly, not that familiar with it), so I just edited an swf.

Reply:
Alright, that's fine. I'll mark this as resolved.